shared/actions/config/index.tsx

```diff
@@ -102,6 +110,10 @@ function* loadDaemonBootstrapStatus(
     yield Saga.put(loadedAction)
     // request follower info in the background
     yield RPCTypes.configRequestFollowerInfoRpcPromise({uid: s.uid})
+    // set HTTP srv info
+    yield Saga.put(
+      ConfigGen.createUpdateHTTPSrvInfo({address: s.httpSrvInfo.address, token: s.httpSrvInfo.token})
```

Comment:
You don't need to make this. Just handle it in the reducer

Reply:
I did it like this because this action also gets fired as a result of a notification from the service. With this, there is just one place in the reducer where these values are updated, instead of the bootstrap handler doing it, and then a handler for the notification. Seemed cleaner this way.